### PR TITLE
Add "gh repo create" to SETUP.md

### PR DESCRIPTION
**Pull request by lubianat**
As discussed in issue #418 .

### CODE REVIEW

File: SETUP.md [hunk 1]
```diff
@@ -63,7 +63,11 @@ git remote set-url origin https://github.com/$OWNER/$REPO.git
 git remote set-url origin git@github.com:$OWNER/$REPO.git
 ```
 
-Next, you must manually create an empty GitHub repository at <https://github.com/new>.
+Then you need to create an empty GitHub repository. 
+You can do that at <https://github.com/new> or via the  [GitHub command line interface](https://github.com/cli/cli):
```
Comment (dhimmel) [inline SETUP.md:67]:
```suggestion
Then create an empty repository on GitHub. 
You can do this at <https://github.com/new> or via the [GitHub command line interface](https://github.com/cli/cli) (if installed) with `gh repo create`.
```

If accepting this suggestion, delete the following code block

File: SETUP.md [hunk 2]
```diff
@@ -63,7 +63,11 @@ git remote set-url origin https://github.com/$OWNER/$REPO.git
...
+```sh
+$ gh repo create
```
Comment (dhimmel) [inline SETUP.md:69]: we tend not to prefix terminal commands with `$`, since it makes it harder to copy paste the entire code block

**Review by dhimmel — COMMENTED** [review]
Thanks for the documentation support.

I suggest a minor tweak of the wording (feel free to accept parts or none of it).

File: SETUP.md [hunk 3]
```diff
@@ -63,7 +63,11 @@ git remote set-url origin https://github.com/$OWNER/$REPO.git
 git remote set-url origin git@github.com:$OWNER/$REPO.git
 ```
 
-Next, you must manually create an empty GitHub repository at <https://github.com/new>.
+Then create an empty repository on GitHub. 
+You can do this at <https://github.com/new> or via the [GitHub command line interface](https://github.com/cli/cli) (if installed) with `gh repo create`.
+```sh
+$ gh repo create
+```
```
Comment (agitter) [inline SETUP.md:70]:
```suggestion
```
This code block can be removed now that the command is inline.

**Review by agitter — APPROVED** [review]
We're all in agreement about what to add here so I pushed the suggested modification.

**Pull request merged**